sdk/cognitivelanguage/ai-language-text/src/models.ts

```diff
@@ -774,8 +913,10 @@ export interface CustomActionMetadata {
 /**
  * The state of a succeeded batched action.
  */
-export interface BatchActionSuccessResult<T, Kind extends AnalyzeBatchActionName>
-  extends BatchActionState<Kind> {
+export interface BatchActionSuccessResult<
+  T extends DocumentDetectedLanguage,
```

Comment:
Is this a breaking change? Is T used to be anything, but now it can be only a specific type that extends DocumentDetectedLanguge?

Reply:
Good question, yes it is a breaking change if the customer is using `BatchActionSuccessResult` with any type they like but this is not a scenario we're supporting. `BatchActionSuccessResult` is an output model and this change says that the model now has an extra field described by `DocumentDetectedLanguage`.